addons/project/static/tests/views/project_task_kanban/project_task_kanban_view_tests.js

```diff
@@ -15,19 +15,19 @@ let target;
 
 QUnit.module('Project', {
     beforeEach: async function () {
-        const pyEnv = await startServer();
-        const projectId = pyEnv['project.project'].create([
+        this.pyEnv = await startServer();
```

Comment:
no need to put `pyEnv` in `this`, we could use the hook `getPyEnv` to get it inside our test cases.

imported like this: 

```js
import { getPyEnv } from '@bus/../tests/helpers/mock_python_environment';
```